collections/Add_A/MS_Add_A_188.xml

```diff
@@ -53,122 +57,180 @@
                      </binding>
                   </bindingDesc>
                </physDesc>
+               <history>
+                  <origin resp="#PA">
+                     <origDate calendar="Gregorian" notAfter="1150" notBefore="1125">12th century, second quarter</origDate>
+                     <origPlace>
+                        <country key="place_7000084">German</country>, <region>West</region>
+                        <!--ORIGINAL: German, West-->
+                     </origPlace>
+                  </origin>
+                  <provenance resp="#VD">Regular canons of <orgName key="org_145" role="fmo">St Mary and St James, Lahneck</orgName> (slightly south of Koblenz), who moved in 1326 to Mayen (west of Koblenz); with 15th century inscription: <q>Notum sit omnibus presens scriptum visuris in perpetuum quod iste liber est ecclesie sancte Marie virginis sanctique Iacobi apostoli Lůnecke quem si quis furto detraxerit aut in vadio exposuerit …</q>, and in a 17th(?)-century hand, <q>1326. Hic liber cum caeteri Maioniam <note>[Mayen]</note> translatus sub Baldewino archiepiscopo.</q>(fol.167v) </provenance>
+                  <acquisition when="1879">Thomas Boone, London bookseller, with his price-code in pencil (fol. 168r, upper left corner); bought by the Bodleian on 29 October 1879 for £10. Former Bodleian shelfmarks: ‘MSS. Bodl. Addit. A. 188’ (fol. iir).</acquisition>
+               </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>Adapted (2018) from S. J. P. Van Dijk, <title>Handlist of the Latin Liturgical Manuscripts in the Bodleian Library Oxford : Vol. 1: Mass Books</title> (typescript, 1957), Pächt and Alexander, and the Summary Catalogue (1905); with description of decoration by Elizabeth Solopova, c. 2000. <listBibl>
-                              <bibl facs="aam0607.gif" type="SC">Summary Catalogue, Vol. 5, p. 572</bibl>
-                           </listBibl>
-                        </source>
+                        <source>Description by Peter Kidd (June 2021). For previous descriptions see Bibliography, below.
+                           <!-- <bibl facs="aam0607.gif" type="SC">Summary Catalogue V, p. 572</bibl> --></source>
                      </recordHist>
                   </adminInfo>
                   <surrogates><bibl subtype="partial" type="digital-facsimile"><ref target="https://digital.bodleian.ox.ac.uk/objects/2c34aeef-72ae-4a29-9f29-13d438c623f7/"><title>Digital Bodleian</title></ref>
                         <note>(1 image from 35mm slides)</note></bibl></surrogates>
                   <listBibl type="WRAPPER">
                      <listBibl>
-                        <bibl>Otto Pächt and J. J. G. Alexander, <title>Illuminated Manuscripts in the Bodleian Library Oxford</title>, I (1966), no. 55; II (1970), no. 122</bibl>
+                        <head>Bodleian Library catalogues</head>
```

Comment:
Is Van Dijk technically a Bodleian catalogue? Personally I would leave out the heading.